app/src/main/java/com/example/a310_rondayview/SignInActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.example.a310_rondayview;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class SignInActivity extends AppCompatActivity {
+
+    private FirebaseAuth mAuth;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
Great work on the implementation here, if I could just suggest one thing, it would be adding a couple of comments in this method to make it clearer what this snippet is doing :)

Reply:
Just looked at the SignUpActivity, would we potentially also be able to do the same there? Thank you!